openwrt-openwisp-monitoring/files/monitoring.config

```diff
@@ -0,0 +1,3 @@
+config monitoring 'monitoring'
+	option monitored_interfaces '*'
```

Comment:
The `*` is a cool feature.  Can we leverage the implementation somehow to allow for multiple management VPNs too?  Shall I open an issue to track that discussion?

Reply:
We can :thinking: For now, we are taking all vpn interfaces into consideration

Reply:
@nickberry17 management interface information is sent by [openwisp-config](https://github.com/openwisp/openwisp-config)

Reply:
Sorry, I know I'm going off on a tangent, but what I meant is can we define multiple management VPN interfaces for openwisp-config?  I tried adding two (wg0 and tun0) but that did not work.  I'm not asking about monitoring stats from them.  If that is not currently possible then I would like to open an issue for that.

Reply:
@nickberry17 that is not possible at the moment, only 1 management interface is supported